### PR TITLE
Update JDBC trace for Oracle 23c

### PR DESCRIPTION
**Pull request by KyleAure**
Oracle 23c no longer has an ojdbcX_g variant. 
Update our tracing docs to align with this change.

### CODE REVIEW

File: modules/ROOT/pages/jdbc-tracing.adoc [hunk 1]
```diff
@@ -179,16 +179,18 @@ The Microsoft SQL Server DataDirect driver does not support the `java.util.loggi
 [#Oracle]
 === Oracle
 
-Oracle provides two different drivers, one for production and another for debugging purposes. The production driver does not produce any trace, so you need to download and replace your production driver with the debugging driver. The debugging driver has `_g` in the driver name. For example, `ojdbc8.jar` is `ojdbc8_g.jar`. Configure the debugging driver by specifying the `library` subelement within the `jdbcDriver` element. The `library` subelement defines the path to the debugging driver JAR file.
+Prior to Oracle 23c Oracle provided two different drivers, one for production and another for debugging purposes. The production driver does not produce any trace, so you need to download and replace your production driver with the debugging driver. The debugging driver has `_g` in the driver name. For example, `ojdbc8.jar` is `ojdbc8_g.jar`. Configure the debugging driver by specifying the `library` subelement within the `jdbcDriver` element. The `library` subelement defines the path to the debugging driver JAR file.
```
Comment (dmuelle) [inline modules/ROOT/pages/jdbc-tracing.adoc]:
```suggestion
Prior to Oracle 23c, Oracle provided two different drivers, one for production and another for debugging purposes. The production driver does not produce any trace, so you need to download and replace your production driver with the debugging driver. The debugging driver has `_g` in the driver name. For example, `ojdbc8.jar` is `ojdbc8_g.jar`. Configure the debugging driver by specifying the `library` subelement within the `jdbcDriver` element. The `library` subelement defines the path to the debugging driver JAR file.
```

File: modules/ROOT/pages/jdbc-tracing.adoc [hunk 2]
```diff
@@ -179,16 +179,18 @@ The Microsoft SQL Server DataDirect driver does not support the `java.util.loggi
...
 
-The Oracle driver supports the `java.util.logging` library. The driver uses the `oracle` package name for trace specification. The following `server.xml` file example shows you how to configure the Oracle debugging driver and enable trace.
+After Oracle 23c there are no `_g` drivers and diagnotic trace is provided in the base JDBC driver.
```
Comment (dmuelle) [inline modules/ROOT/pages/jdbc-tracing.adoc]:
```suggestion
In Oracle 23c and later, Oracle no longer provides the `_g` drivers and diagnostic trace is provided in the base JDBC driver.
```

**Review by dmuelle — COMMENTED** [review]
ID review: minor suggestions per IBM style

**Comment by KyleAure** [comment]
@dmuelle - Committed your suggested changes

**Review by dmuelle — COMMENTED** [review]
Thanks, LGTM- merging but we'll need to also PR this change to staging from your branch

**Pull request merged**